library/Zend/Loader/ComposerAutoloader.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Loader;
+
+use Composer\Autoload\ClassLoader;
```

Comment:
This couples `Zend\Loader` with the composer autoloader: it's much easier to just use the composer loader directly instead (and avoid coupling just for adapter purposes)